docs/options.md

```diff
@@ -364,3 +364,22 @@ Valid options:
 | Default | CLI Override                                                | API Override                                               |
 | ------- | ----------------------------------------------------------- | ---------------------------------------------------------- |
 | `"lf"`  | <code>--end-of-line <lf&#124;crlf&#124;cr&#124;auto></code> | <code>endOfLine: "<lf&#124;crlf&#124;cr&#124;auto>"</code> |
+
+## Embedded Language Formatting
+
+_First available in v2.1.0_
+
+Control whether Prettier formats quoted code embedded in the file.
+
+When Prettier identifies cases where it looks like you've placed some code it knows how to format within a string in another file, like in a tagged template in JavaScript with a tag named `html` or in code blocks in Markdown, it will by default try to format that code.
+
+Sometimes this behavior is undesirable, particularly in cases where you might not have intended the string to be interpreted as code. This option allows you to switch between the default behavior (`auto`) and disabling this feature entirely (`off`).
+
+Valid options:
+
+- `"auto"` – Format embedded code if Prettier can automatically identify it.
+- `"off"` - Never automatically format embedded code.
+
+| Default  | CLI Override                                    | API Override                                   |
+| -------- | ----------------------------------------------- | ---------------------------------------------- |
+| `"auto"` | <code>--embedded-language-formatting=off</code> | <code>embeddedLanguageFormatting: "off"</code> |
```

Comment:
```suggestion
| `"auto"` | `--embedded-language-formatting=off` | `embeddedLanguageFormatting: "off"` |
```

Reply:
Done, though there's a bunch of other usages of `<code>` in this file.

Reply:
Fixed in #7937